tests/test_base.py

```diff
@@ -31,12 +31,17 @@ def teardown_method(self, method):
         self.conn.flushdb()
 
     def _ensure_queues(self, queued=None, active=None, error=None,
-                       scheduled=None):
+                       scheduled=None, ignore_queues=[]):
```

Comment:
Maybe we could only register the periodic task in a certain test class so we don't have to include this check everywhere?